### DISCUSSION TITLE
Error connection refused on running command php artisan sentry:test

### DISCUSSION THREAD

**Discussion by Jhon-Henkel** [getsentry/sentry-laravel#704] [Q&A] [answered]
On running command on server by ssh, return this error:

There was an error sending the event.
SDK: Failed to send the event to Sentry. Reason: "Failed to connect to [URL] port 443: Connection refused for "[URL]/envelope/".".
Please check the error message from the SDK above for further hints about what went wrong.

ps.: [URL] is dsn on .env file.

**Comment by Jhon-Henkel** [comment]
In localhost error is sending to sentry success

**Comment by stayallive** [comment]
Are you inside a corporate network and/or does the server have a restricted outgoing connection policy? 

If the same DSN works on your local machine my guess is that the outgoing connection is not allowed by a firewall rule on the server. You probably have to configure a HTTP proxy or allow the connection to that URL from your server.

**Reply by Jhon-Henkel** [reply]
My server is a shared server VPS,the support of service said don't have any rule to block 443 port.

I haven't gotten results yet, but I'll keep trying and as soon as I get something I'll post here

**Comment by Jhon-Henkel** [comment]
Today, i test runing curl for https://sentry.io inside server per ssh command.
Return of this command is curl: (7) Failed to connect to sentry.io port 443: Connection refused.
Running curl for http://sentry.io is done with success, but test with dsn with http dont work for me to.

**Comment by Jhon-Henkel** [comment]
If i pull of letter "s" from https on .env file, return this error on test:
Sending test event...
There was an error sending the event.
Please check if your DSN is set properly in your `.env` as `SENTRY_LARAVEL_DSN` or in your config file `config/sentry.php`.

**Comment by Jhon-Henkel** [comment, marked as answer]
The problem is on connection to sentry, sentry block connections from inside my server. 
Solution for my case is migrate for other platform. I implements the honey badger. Now is working correctly.

**Reply by Jhon-Henkel** [reply]
In research I found that Sentry blocks connections to arc linux, my VPS uses an arc based distribution. Hence all these problems. The "solution" even in this case is to use another platform or migrate server.

**Reply by stayallive** [reply]
We don't block specific connections from specific OS'es, it is more likely a misconfiguration on your server or firewall active on the server hence why another platform or server might work just fine.

If you do believe you are blocked please contact Sentry support with the relevant IP addresses and other pertinent details of your server so they can take a look from their side and confirm that you are not blocked!
